### PR TITLE
Feat(CX-3494): Implement Tracking in new SWA modules

### PR DESCRIPTION
**Pull request by lordkiz**
This PR resolves [CX-3494] 

### Description
- This PR adds consign and inquiry tracking on the CTAs in the new SWA Landing page

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Implement Tracking in new SWA modules - kizito

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-3494]: https://artsyproduct.atlassian.net/browse/CX-3494?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/SellWithArtsy/Components/Header.tsx [hunk 1]
```diff
@@ -43,13 +34,15 @@ export const Header: React.FC<HeaderProps> = ({ onConsignPress, onInquiryPress }
...
-}) => {
+const OldHeader: React.FC<{
+  handleInquiryPress: () => void
+  handleSubmitPress: (subject: string) => void
+}> = ({ handleInquiryPress, handleSubmitPress }) => {
   const screenDimensions = useScreenDimensions()
 
   const enableInquiry = useFeatureFlag("AREnableConsignmentInquiry")
+
+  const buttonText = "Submit an Artwork"
```
Comment (dariakoko) [inline src/app/Scenes/SellWithArtsy/Components/Header.tsx:45]: Shouldn't it be "Start Selling"?

Comment (dariakoko) [reply src/app/Scenes/SellWithArtsy/Components/Header.tsx:45]: Ah no, it shouldn't, that's an old component. Apologies

**Review by dariakoko — DISMISSED** [review]
Looks good, thank you!

File: src/app/Scenes/SellWithArtsy/Components/Footer.tsx [hunk 2]
```diff
@@ -68,20 +67,31 @@ const NewFooter: React.FC<{
...
+  const buttonText = "Start Selling"
   return (
     <Flex mx={2} alignItems={isTablet ? "center" : undefined}>
       <Text variant="lg-display">
         Sell with Artsy is the simple, contemporary way to sell art from your collection.
       </Text>
       <Spacer y={2} />
-      {/**TODO: Implement tracking. Add sellHowItWorks to ContextModule */}
       <Button
+        testID="Footer-consign-CTA"
```
Comment (olerichter00) [inline src/app/Scenes/SellWithArtsy/Components/Footer.tsx:78]: **suggestion:** To keep it consistent.

```suggestion
        testID="footer-consign-CTA"
```

File: src/app/Scenes/SellWithArtsy/Components/Header.tsx [hunk 3]
```diff
@@ -124,10 +126,24 @@ const NewHeader: React.FC<{ handleInquiryPress: () => void; handleSubmitPress: (
           for your work.
         </Text>
         <Flex justifyContent="center" alignItems="center">
-          <Button block onPress={handleSubmitPress} my={1} variant="fillDark">
-            Start Selling
+          <Button
+            testID="Header-consign-CTA"
```
Comment (olerichter00) [inline src/app/Scenes/SellWithArtsy/Components/Header.tsx:130]:
```suggestion
            testID="header-consign-CTA"
```

File: src/app/Scenes/SellWithArtsy/Components/Header.tsx [hunk 4]
```diff
@@ -103,10 +104,11 @@ const OldHeader: React.FC<{ handleInquiryPress: () => void; handleSubmitPress: (
   )
 }
 
-const NewHeader: React.FC<{ handleInquiryPress: () => void; handleSubmitPress: () => void }> = ({
-  handleInquiryPress,
-  handleSubmitPress,
-}) => {
+const NewHeader: React.FC<{
```
Comment (olerichter00) [inline src/app/Scenes/SellWithArtsy/Components/Header.tsx:107]:
```suggestion
const Header: React.FC<{
```

Comment (dariakoko) [reply src/app/Scenes/SellWithArtsy/Components/Header.tsx:107]: Here we have two headers - old and new. We'll just refactor this part when we remove the feature flag logic

Comment (olerichter00) [reply src/app/Scenes/SellWithArtsy/Components/Header.tsx:107]: Ah, and both are in the same file, I guess..

**Review by olerichter00 — APPROVED** [review]

File: src/app/Scenes/SellWithArtsy/Components/Header.tsx (hunk 4 above)
Comment (dariakoko) [reply src/app/Scenes/SellWithArtsy/Components/Header.tsx:107]: Yesss

**Pull request merged**